Formula/yarn.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class Yarn < Formula
+  desc "Javascript package manager"
+  homepage "https://yarnpkg.com/"
+  url "https://registry.npmjs.org/yarn/-/yarn-0.15.1.tgz"
+  sha256 "f99fd587e84987909d5f9e918b8fe524349fdc548e5bc5c380c8f8c0a70c6b87"
+  head "https://github.com/yarnpkg/yarn.git"
+
+  depends_on "node"
```

Comment:
Why does it depend on node? A lot of people probably use `nvm` and therefore do not want the `node` package from brew. 
Can we remove that or have a check to see if node is already available in the current shell?